src/docs/user/references/SchedulerGraphQLAPI.adoc

```diff
@@ -0,0 +1,380 @@
+      endCursor
+    }
+  }
+}
+----
+Which returns a result such as:
+[source,json]
+----
+{
+  "data": {
+    "jobs": {
+      "edges": [
+        {
+          "node": {
+            "id": "6"
+          },
+          "cursor": "Z3JhcGhxbC1jdXJzb3I2"
+        },
+        {
+          "node": {
+            "id": "7"
+          },
+          "cursor": "Z3JhcGhxbC1jdXJzb3I3"
+        },
+        {
+          "node": {
+            "id": "8"
+          },
+          "cursor": "Z3JhcGhxbC1jdXJzb3I4"
+        },
+        {
+          "node": {
+            "id": "9"
+          },
+          "cursor": "Z3JhcGhxbC1jdXJzb3I5"
+        },
+        {
+          "node": {
+            "id": "10"
+          },
+          "cursor": "Z3JhcGhxbC1jdXJzb3IxMA=="
+        }
+      ],
+      "pageInfo": {
+        "hasNextPage": true,
+        "hasPreviousPage": true,
+        "startCursor": "Z3JhcGhxbC1jdXJzb3I2",
+        "endCursor": "Z3JhcGhxbC1jdXJzb3IxMA=="
+      }
+    }
+  }
+}
+----
+
+===== GraphQL Filters
+Graphql allows to select jobs according to various *filters*. The following example filters job according to their name:
+[source,graphql]
+----
+{
+  jobs(filter: {name: "*Python"}) {
+    edges {
+      node {
+        id name
+      }
+    }
+  }
+}
+----
+The wildcard "\*Python" selects all jobs ending by "Python". The special character `*` can be used at the beginning or at the end of the expression (or both). It cannot be used in the middle.
+
+Here are the results of this GraphQL query:
+[source,json]
+----
+{
+  "data": {
+    "jobs": {
+      "edges": [
+        {
+          "node": {
+            "id": "8",
+            "name": "Script_Python"
+          }
+        },
+        {
+          "node": {
+            "id": "9",
+            "name": "Script_Python"
+          }
+        },
+        {
+          "node": {
+            "id": "10",
+            "name": "Script_Python"
+          }
+        },
+        {
+          "node": {
+            "id": "11",
+            "name": "Script_Python"
+          }
+        }
+      ]
+    }
+  }
+}
+----
+The complete list of filters is available in the GraphQL `Docs` section.
+
+====  Scheduler GraphQL REST API
+
+After testing queries in the GraphQL Portal, you can use the scheduling-api REST interface to run these queries from any REST client.
+
+There are a few aspects to consider when running graphql queries from a REST client:
+
+ * The url of the graphql REST api is different: `http://your-server:8080/scheduling-api/v1/graphql` (_graphql_ instead of graphiql)
+ * The REST request method is `POST`, its `Content-Type` is `application/json`
+ * It requires in the header section, a `sessionid` retrieved from a login to the ProActive scheduler REST API.
+ * The query itself is given as a string value of the `"query"` attribute inside the json body content. This value must be *escaped* to be a valid json. You can use, for example, online tools to escape the query as a json string.
```

Comment:
'You can use, for example, online tools to escape the query as a json string.' => Is this worthy to be mentioned ? To me, this doesn't provide much info.

Reply:
Yes, I think it is useful.